definitions/subworkflows/cnvkit_single_sample.cwl

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "Subworkflow that runs cnvkit in single sample mode and returns a vcf file"
+
+inputs:
+    tumor_bam:
+        type: File
+    method:
+        type: string?
+        default: "hybrid"
+    diagram:
+        type: boolean?
+    scatter_plot:
+        type: boolean?
+    drop_low_coverage:
+        type: boolean?
+    male_reference:
+        type: boolean?
+    reference_cnn:
+        type: File
+    cnvkit_vcf_name:
+        type: string?
```

Comment:
Could be useful to have a default of something like:
`$(inputs.tumor_bam.nameroot).cnvkit.vcf`

Reply:
Good idea, added in ` [957435e](/genome/analysis-workflows/pull/556/commits/957435e34fd7f3c50a6289146797522c22531f8a)`